### ISSUE TITLE
SQL parsing error

### ISSUE THREAD

**Issue by apla** [pekim/tedious#347]
statement:

```sql
SELECT
     TABLE_NAME = t.name,
     COLUMN_NAME = ind.name,
     IndexId = ind.index_id,
     ColumnId = ic.index_column_id,
     ColumnName = col.name,
     CASE WHEN ind.is_primary_key = 1 THEN 'PRIMARY' WHEN ind.is_unique = 1 THEN 'UNIQUE' ELSE '' END as INDEX_NAME,
--     IS_PRIMARY_KEY_ = ind.is_primary_key,
--     IS_UNIQUE_ = ind.is_unique,
     ind.*,
     ic.*,
     col.*
FROM
     sys.indexes ind
INNER JOIN
     sys.index_columns ic ON  ind.object_id = ic.object_id and ind.index_id = ic.index_id
INNER JOIN
     sys.columns col ON ic.object_id = col.object_id and ic.column_id = col.column_id
INNER JOIN
     sys.tables t ON ind.object_id = t.object_id
WHERE
     t.is_ms_shipped = 0
ORDER BY
     t.name, ind.name, ind.index_id, ic.index_column_id;
```

error:

```
Uncaught TypeError: Cannot call method 'toString' of null
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:399:19
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:406:14
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:405:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readBuffer (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:402:12)
      at readMax (/usr/local/lib/node_modules/tedious/lib/value-parser.js:404:10)
      at readMaxNChars (/usr/local/lib/node_modules/tedious/lib/value-parser.js:398:3)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:199:20
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:82:16)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:100:25
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:86:14)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:197:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readInt32LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:194:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:103:25
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:86:14)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:183:17
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:182:29
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:70:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:374:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:405:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readBuffer (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:402:12)
      at readNChars (/usr/local/lib/node_modules/tedious/lib/value-parser.js:373:19)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:201:20
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:175:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt16LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:172:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:73:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:142:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt8 (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:139:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:100:25
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:86:14)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:197:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readInt32LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:194:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:103:25
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:86:14)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:374:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:405:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readBuffer (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:402:12)
      at readNChars (/usr/local/lib/node_modules/tedious/lib/value-parser.js:373:19)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:201:20
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:175:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt16LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:172:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:73:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:197:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readInt32LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:194:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:103:25
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:86:14)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:364:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:405:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readBuffer (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:402:12)
      at readChars (/usr/local/lib/node_modules/tedious/lib/value-parser.js:363:19)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:193:20
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:175:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt16LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:172:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:73:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:374:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:405:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readBuffer (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:402:12)
      at readNChars (/usr/local/lib/node_modules/tedious/lib/value-parser.js:373:19)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:201:20
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:175:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt16LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:172:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:73:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:197:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readInt32LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:194:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:103:25
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:86:14)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:197:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readInt32LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:194:12)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:103:25
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:86:14)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:374:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:405:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readBuffer (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:402:12)
      at readNChars (/usr/local/lib/node_modules/tedious/lib/value-parser.js:373:19)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:201:20
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:175:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt16LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:172:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:73:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at /usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:43:7
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:374:7
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:405:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readBuffer (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:402:12)
      at readNChars (/usr/local/lib/node_modules/tedious/lib/value-parser.js:373:19)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:201:20
      at /usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:175:9
      at Parser.awaitData (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:116:9)
      at Parser.readUInt16LE (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:172:12)
      at readDataLength (/usr/local/lib/node_modules/tedious/lib/value-parser.js:73:27)
      at /usr/local/lib/node_modules/tedious/lib/value-parser.js:94:5
      at readTextPointerNull (/usr/local/lib/node_modules/tedious/lib/value-parser.js:46:5)
      at exports.default (/usr/local/lib/node_modules/tedious/lib/value-parser.js:93:3)
      at next (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:27:34)
      at Object.exports.default [as 209] (/usr/local/lib/node_modules/tedious/lib/token/row-token-parser.js:47:3)
      at Parser.parseTokens (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:92:29)
      at Parser._transform (/usr/local/lib/node_modules/tedious/lib/token/stream-parser.js:67:14)
      at Parser.Transform._read (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_transform.js:172:10)
      at Parser.Transform._write (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_transform.js:160:12)
      at doWrite (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_writable.js:333:12)
      at writeOrBuffer (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_writable.js:319:5)
      at Parser.Writable.write (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_writable.js:246:11)
      at Parser.addBuffer (/usr/local/lib/node_modules/tedious/lib/token/token-stream-parser.js:59:26)
      at Connection.sendDataToTokenStreamParser (/usr/local/lib/node_modules/tedious/lib/connection.js:684:37)
      at Connection.data (/usr/local/lib/node_modules/tedious/lib/connection.js:1210:21)
      at Connection.dispatchEvent (/usr/local/lib/node_modules/tedious/lib/connection.js:559:45)
      at MessageIO.<anonymous> (/usr/local/lib/node_modules/tedious/lib/connection.js:476:16)
      at ReadablePacketStream.<anonymous> (/usr/local/lib/node_modules/tedious/lib/message-io.js:101:13)
      at readableAddChunk (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_readable.js:199:16)
      at ReadablePacketStream.Readable.push (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_readable.js:163:10)
      at ReadablePacketStream.Transform.push (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_transform.js:133:32)
      at ReadablePacketStream._transform (/usr/local/lib/node_modules/tedious/lib/message-io.js:68:16)
      at ReadablePacketStream.Transform._read (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_transform.js:172:10)
      at ReadablePacketStream.Transform._write (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_transform.js:160:12)
      at doWrite (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_writable.js:333:12)
      at writeOrBuffer (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_writable.js:319:5)
      at ReadablePacketStream.Writable.write (/usr/local/lib/node_modules/tedious/node_modules/readable-stream/lib/_stream_writable.js:246:11)
      at write (_stream_readable.js:602:24)
      at flow (_stream_readable.js:611:7)
      at Socket.pipeOnReadable (_stream_readable.js:643:5)
      at emitReadable_ (_stream_readable.js:427:10)
      at emitReadable (_stream_readable.js:423:5)
      at readableAddChunk (_stream_readable.js:166:9)
      at Socket.Readable.push (_stream_readable.js:128:10)
      at TCP.onread (net.js:529:21)
```

**Comment by bretcope** [comment]
Could you provide a complete example with code, not just the SQL?

**Comment by arthurschreiber** [comment]
I think this was fixed with #340, so I'm closing this for now. Please test again once I pushed a new version to npm. Happy Holidays!
